tasks/python-setup.yaml

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
I think we don't need to include this in this role.

Reply:
I'm adding this is maybe in some case, we need to run python to do the locking and unlocking mechanism

Reply:
so far we don't need python